### PR TITLE
[linux-6.6.y] Add support for Zhaoxin I2C controller

### PR DESCRIPTION
**Pull request by leoliu-oc**
zhaoxin inclusion
category: feature

-------------------

Zhaoxin I2C Linux driver support all bidirectional bus protocols speed specified in the I2C Specification 7.0.

### CODE REVIEW

File: drivers/i2c/busses/Kconfig [hunk 1]
```diff
@@ -334,6 +334,18 @@ config I2C_VIAPRO
 	  This driver can also be built as a module.  If so, the module
 	  will be called i2c-viapro.
 
+config I2C_ZHAOXIN
+	tristate "Zhaoxin I2C controller driver"
+	depends on PCI
+	default m
```
Comment (opsiff) [inline drivers/i2c/busses/Kconfig:340]: 为什么要default m,arm平台上也需要默认打开这个？

**Review by opsiff — APPROVED** [review]
/ok-to-test
/lgtm

File: drivers/i2c/busses/Kconfig (hunk 1 above)
Comment (leoliu-oc) [reply drivers/i2c/busses/Kconfig:340]: 这边会修改为依赖CONFIG_CPU_SUP_ZHAOXIN

**Pull request merged**